frontend-site/src/modules/projectValues.js

```diff
@@ -1,4 +1,4 @@
-export const statusFilterValues = Object.freeze(['complete', 'incomplete', 'abandoned']);
+export const statusFilterValues = Object.freeze(['complete', 'active', 'inActive']);
```

Comment:
Should we keep all the values lowercase (i.e. `inactive` versus `inActive`)?

Reply:
`inActive` to me implies two words, like "in active", which isn't what we intend.

Reply:
sorry about that made the change from my phone I'll get that fixed